frontend/packages/operator-lifecycle-manager/integration-tests/scenarios/global-installmode.scenario.ts

```diff
@@ -98,6 +98,7 @@ describe('Interacting with an `AllNamespaces` install mode Operator (Jaeger)', (
     await catalogView.categoryTabsPresent();
     await catalogView.categoryTabs.get(0).click();
     await catalogPageView.clickFilterCheckbox(customProviderUID);
+    await browser.wait(until.visibilityOf(catalogPageView.catalogTileByID(jaegerTileID)));
```

Comment:
I recommend putting the `browser.wait' into `catalogPageView.catalogTileByID` as there are other tests which use the `catalogPageView.clickFilterCheckbox(customProviderUID)` then `catalogPageView.catalogTileByID(jaegerTileID)`.
Or.  add the 'browser.wait' to the following tests as well:
- displays subscription creation form for selected Operator
- displays Operator as subscribed in OperatorHub
- displays subscription creation form for selected Operator

Reply:
We don't want it in `catalogPageView.catalogTileByID`. This logic is specific to a particular test and does not belong in a view file. It doesn't fit the scenario/view separation of concerns.

There is one other place I could add a wait. I'll push an update.

Reply:
Putting it in the view file would also prevent us from writing tests to check that a tile is not displayed (e.g. when a certain filter is selected).

Reply:
Ok, sounds good.  I'd add it to the other tests which need it.